extension/manifest.json

```diff
@@ -36,6 +36,7 @@
             "controller/translation/Translation.js",
             "controller/translation/TranslationMessage.js",
             "controller/translation/translationWorker.js",
+            "controller/translation/bergamotTranslatorVersion.js",
```

Comment:
Is there a tab here? Just remove it please

Reply:
done